src/main/java/seedu/task/logic/commands/HelpCommand.java

```diff
@@ -9,18 +9,35 @@
  */
 public class HelpCommand extends Command {
 
-    public static final String COMMAND_WORD = "help";
-
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-            + "Example: " + COMMAND_WORD;
-
-    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
-
-    public HelpCommand() {}
-
-    @Override
-    public CommandResult execute() {
-        EventsCenter.getInstance().post(new ShowHelpEvent());
-        return new CommandResult(SHOWING_HELP_MESSAGE);
```

Comment:
Apparently u forgot to change the SHOWING_HELP_MESSAGE in the HelpCommandTest file.